ingestion/src/metadata/utils/s3_utils.py

```diff
@@ -90,11 +90,13 @@ def read_parquet_from_s3(client: Any, key: str, bucket_name: str):
     """
     Read the parquet file from the s3 bucket and return a dataframe
     """
-    s3_fs = s3fs.S3FileSystem(
-        key=client.awsAccessKeyId,
-        secret=client.awsSecretAccessKey.get_secret_value(),
-        token=client.awsSessionToken,
-    )
+    s3_fs = s3fs.S3FileSystem()
+    if client.awsAccessKeyId and client.awsSecretAccessKeyx:
+        s3_fs = s3fs.S3FileSystem(
+            key=client.awsAccessKeyId,
+            secret=client.awsSecretAccessKey.get_secret_value(),
+            token=client.awsSessionToken,
+        )
```

Comment:
we might also need a check for awsSessionToken

Reply:
I tested out this scenario `client.awsSessionToken` is by default set to `None` and it is okay for `token` value to be `None`